Profile Folder/chrome/locale/ja/properties/gsettings.properties

```diff
@@ -1,15 +1,15 @@
-geckiumSettings=[!!! Gèçƙïú₥ §èƭƭïñϱƨ !!!]
+geckiumSettings=Geckium の設定
 
-addAnApp=Add an app
+addAnApp=アプリを追加する
 
-themeHasNoDescription=This theme has no description.
-builtInTheme=[!!! [ßÛÌ£T-ÌN] {{themeName}} !!!]
-showInFolder=[!!! §λôω ïñ ƒôℓδèř !!!]
-apply=[!!! Âƥƥℓ¥ !!!]
-yes=[!!! Ýèƨ !!!]
+themeHasNoDescription=このテーマには概要がありません。
+builtInTheme=[ビルトイン] {{themeName}}
```

Comment:
Is this change fine with you? I had to add brackets to separate `BUILT-IN` from the theme name, and I also had to add `{{themeName}}` for the theme name to show up.

Reply:
yes